frontend/package.json

```diff
@@ -106,6 +100,7 @@
     "@types/yup": "0.26.32",
     "axios-mock-adapter": "1.17.0",
     "cross-env": "7.0.2",
+    "node-sass": "4.14.1",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.2",
     "enzyme-to-json": "3.4.4",
```

Comment:
The png file below wasn't compressed before including it in pims, so it was huge.